ansible_collections/arista/avd/molecule/eos_designs-mpls-isis-sr-ldp/inventory/group_vars/SITE1_CONNECTED_ENDPOINTS.yml

```diff
@@ -17,10 +17,11 @@ cpes:
         switches:
           - SITE1-LER1
           - SITE1-LER2
+        ethernet_segment:
+          short_esi: 0303:0202:0101
         port_channel:
           description: EVPN-A-A-PortChannel
           mode: active
-          short_esi: 0303:0202:0101
           subinterfaces:
             - number: 111
               short_esi: 0303:0202:0111
```

Comment:
is this still supported? (short_esi under subinterfaces ?)

If so it should probably be tacked in this PR as well!

Reply:
I am not sure about it, I didn't see anything in comments/docs regarding its deprecation.
For now, I have added `subinterfaces.short_esi` inside eos_designs_deprecated_vars input data-model.

Please let me know if anything else needs to be updated for subinterfaces.short_esi

Reply:
`subinterfaces.short_esi` is _not_ deprecated (yet)